### DISCUSSION TITLE
CarPlay won't work

### DISCUSSION THREAD

**Discussion by MuchaZ** [Mr-MIBonk/M.I.B._More-Incredible-Bash#25] [Q&A]
Hi,

After the installation of the latest software and applying the patch, I cannot make CarPlay to work with my iPhone 13 Pro. After clicking on Smartphone interface in the connection manager, nothing happens, as if this function couldn't be selected at all.

On top of that, while playing with MIB functions, I made my car's miniature in the menu dissappear.also the Navi and traffic signs recognition stopped working. MIB is very small now. 
![image](https://user-images.githubusercontent.com/5638490/183345215-05cb8cf0-afe0-4503-9653-bbd7ab40023b.jpeg)
![image](https://user-images.githubusercontent.com/5638490/183345285-009f7349-3ff9-4802-b16b-9bbec6754f74.jpeg)

I would be glad for any help. :)

The car is 2017 Audi A5. It was bought with Carplay, but after some time stopped to work. I thought this software would be a solution.

**Comment by CaneTLOTW** [comment]
Seems that your unit is alread patched. The checksum at the end of the file name does not fir.
It should be MU1438-ifs-root-part2-0x00ba0000-7899EE00.ifs
But it is also not the M.I.B patch which has 8099EE00.

Safe way is, that you apply standard FW on unit and patch afterwards.

**Reply by MuchaZ** [reply]
Thanks for a reply. How to apply the standard FW? I assume it's one of the files from the patches/ directory and should be performed via the red menu?

**Reply by CaneTLOTW** [reply]
No, that would be a normal FW update. Installed via REM.

**Reply by MuchaZ** [reply]
@CaneTLOTW I tried to do it from the normal menu as well as from the red one without a success, cause I'm obviously on the newest version already (3346). Is there a way to force-update and override everything? If not, what else could I do?

**Reply by harman-f** [reply]
SWDL has to be activated in GEM 1st, to enable you to select user defined update in REM.
Or you go for a customized metainfo.txt
[metainfo2.txt](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/9331029/metainfo2.txt)
![grafik](https://user-images.githubusercontent.com/98130152/184474189-85ef935c-0fd7-4669-a802-a04c05c7cf05.png)

**Reply by MuchaZ** [reply]
Thank you @harman-f. The custom metadata enabled to force update all packages (with some problems with finding tuner's files), so I skipped it). CarPlay is working now, however I'm still missing the car's icon in the menu and the nivigation is still not able to load itself (hanging at 11%). Any further advice?

**Comment by eyeblve** [comment]
The same thing happened to me as to you, you have a misconfigured 5F module, I had to go to audi to solve it.

**Reply by eyeblve** [reply]
SVM fix with engine on? I think my mistake was that
